### DISCUSSION TITLE
Welcome to fastly-dashboards Discussions!

### DISCUSSION THREAD

**Discussion by mrnetops** [mrnetops/fastly-dashboards#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by john-lauro** [comment]
Thank you for sharing this repo. 
[mrnetops.txt](https://github.com/mrnetops/fastly-dashboards/files/8445054/mrnetops.txt)

I have this running locally, however the data is not populating. Thoughts on what I might want to look for? I have the Fastly API Token saved as both an env variable and in a .env file.
